examples/global-boundaries.ipynb

```diff
@@ -0,0 +1,648 @@
+    "Let's take a look at the Boundaries GeoDataFrame"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "id": "322fba19-e0ef-4535-a5cd-97d564315164",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>geometry</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>MULTILINESTRING ((-88.77009 30.24290, -88.7527...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>MULTILINESTRING ((-75.74650 35.19336, -75.7455...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>LINESTRING (-103.00221 37.00011, -103.00253 36...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>MULTILINESTRING ((-75.64425 37.97135, -75.6433...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>LINESTRING (-77.71952 39.32132, -77.75579 39.3...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                            geometry\n",
+       "0  MULTILINESTRING ((-88.77009 30.24290, -88.7527...\n",
+       "1  MULTILINESTRING ((-75.74650 35.19336, -75.7455...\n",
+       "2  LINESTRING (-103.00221 37.00011, -103.00253 36...\n",
+       "3  MULTILINESTRING ((-75.64425 37.97135, -75.6433...\n",
+       "4  LINESTRING (-77.71952 39.32132, -77.75579 39.3..."
+      ]
+     },
+     "execution_count": 38,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "boundary_gdf.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b84c863e-4c4a-430f-ac08-0eb561fcbca5",
+   "metadata": {},
+   "source": [
+    "## Combining SolidPolygonLayer and PathLayer \n",
+    "Now let's create a lonboard `PathLayer` from the `boundary` column of `LineString` and `MultiLineString` geometries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3f4b54a-44b3-4033-b459-37f84df2f49a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set up our fill and line colors\n",
+    "fill_rgb = [204, 251, 254] # light blue\n",
+    "border_rgb = [37, 36, 34] # dark border color"
```

Comment:
Are there any attributes in the data that might make sense for rendering a different color per polygon?

Reply:
The only thing I can think of is... we could possibly render a different color for each `ShapeGroup`.
The shapeGroup attribute is roughly a "country code".
The first features in the FeatureCollection are in `shapeGroup` AFG for Afghanistan.

I think it is pretty cool so quickly render a map of the entire world (like whoa). But I could add some additional logic to show off features like allowing a specific render color based on the "country code"

What do you think?

Reply:
That `ShapeGroup` property seems like the most viable choice to me (there isn't a ton to choose from, is there?) But it would bring a little clarity to the geometries (countries), especially those that are close together in Europe/Middle East.